### PR TITLE
Upgrade to EasyRSA3

### PR DESCRIPTION
**Pull request by lstipakov**
Bundle content of EasyRSA3 Windows release
except openssl binaries. While it downloads content of
64bit release, it also works for win32, since architecture
is relevant only for openssl binaries, which we don't use.

Create EasyRSA-Start.bat, which is used to start
easyrsa shell and adds openvpn's bin directory to the PATH
to make easyrsa use openssl bundled with openvpn.

Signed-off-by: Lev Stipakov <lev@openvpn.net>

### CODE REVIEW

**Comment by lstipakov** [comment]
@TinCanTech could you give it a try, please? 
[OpenVPN-2.5-beta4-I601-amd64.zip](https://github.com/OpenVPN/openvpn-build/files/5256042/OpenVPN-2.5-beta4-I601-amd64.zip)
[OpenVPN-2.5-beta4-I601-x86.zip](https://github.com/OpenVPN/openvpn-build/files/5256045/OpenVPN-2.5-beta4-I601-x86.zip)

**Comment by mattock** [comment]
Looks good but I'm not a WiX expert. I will build-test this tomorrow morning, though.

**Comment by TinCanTech** [comment]
Some initial test results.

Windows7-32bit (VM):
* TAP & Wintun are not listed as installed. And no Start menu items for either.
  Connections work OK,  indicating the devices are installed.

* EasyRSA is still not installed by default.  I can live with that.

* Maintenance options: Modify/Repair are not available from inside installer.

* There is a `res\ovpn.ico` which i was not expecting.

* Might be _better_ to have a Start menu item for `EasyRSA-Start.bat`.

Otherwise, EasyRSA3 works all the way up to _and including_ creating a CA.

**Comment by TinCanTech** [comment]
I would test W10 but .. Running W10 in a VM on my HW is like:

   "Chewing a magnetic ball bearing covered in iron filings"

File: windows-msi/script/Builder.js [hunk 1]
```diff
@@ -2,6 +2,7 @@
  *  openvpn-build — OpenVPN packaging
  *
  *  Copyright (C) 2018-2020 Simon Rozman <simon@rozman.si>
+ *  Copyright (C) 2028-2020 Lev Stipakov <lev@openvpn.net>
```
Comment (becm) [inline windows-msi/script/Builder.js:5]:
```suggestion
 *  Copyright (C) 2020-2020 Lev Stipakov <lev@openvpn.net>
```

**Comment by mattock** [comment]
The "remove static key shortcut" and "add easyrsa 3 shell shortcut" commits look good afaics. If Travis and Appveyor are happy then I'm happy.

**Pull request merged**